src/site/markdown/server/manage-webadmin.md

```diff
@@ -697,6 +697,64 @@ The scheduled task will have the following type `messageId-reindexing` and the f
 
 Warning: During the re-indexing, the result of search operations might be altered.
 
+### Fixing message inconsistencies
+
+This task is only available on top of Guice Cassandra products.
+
+```
+curl -XPOST /messages?task=SolveInconsistencies
+```
+
+Will schedule a task for fixing message inconsistencies between ImapUid and MessageId table. 
```

Comment:
What does it mean for a user? I mean, our users are not supposed the know James cassandra datamodel